### PR TITLE
Block range guarantee - Mercury

### PR DESCRIPTION
**Pull request by vyzaldysanchez**
Goes with: https://github.com/smartcontractkit/chainlink/pull/9022

### CODE REVIEW

**Comment by jmank88** [comment]
Bumping libocr and updating loop here to unblock you: https://github.com/smartcontractkit/chainlink-relay/pull/101

File: .tool-versions [hunk 1]
```diff
@@ -1,2 +1,2 @@
-golang 1.20.1
+golang 1.20.3
```
Comment (jmank88) [inline .tool-versions:1]:
```suggestion
golang 1.20.4
```

File: go.mod [hunk 2]
```diff
@@ -4,52 +4,53 @@ go 1.20
 
 require (
 	github.com/confluentinc/confluent-kafka-go v1.9.2
-	github.com/golang/protobuf v1.5.2
+	github.com/golang/protobuf v1.5.3 // indirect
```
Comment (jmank88) [inline go.mod:7]: Is this tidied? `indirect` is out of place here.

**Review by jmank88 — CHANGES_REQUESTED** [review]
We need to slow down and get all the pieces in to a consistent re-viewable state before merging.

**Review by jmank88 — CHANGES_REQUESTED** [review]
LGTM but we need to wait on libocr, yeah?

**Review by samsondav — APPROVED** [review]

**Review by jmank88 — APPROVED** [review]

**Pull request merged**